selectors/views/lhs.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {GlobalState} from 'types/store';
+import {LhsStaticItem} from 'types/store/lhs';
+
+export function getCurrentLhsStaticItemId(state: GlobalState): string {
```

Comment:
I assume this is written this way because you had some of these in mattermost-redux previously, but could you move these into the same file as the other ones? I'm 0/5 on whether its the one in `/views` or not since we're not really consistent with this folder structure